app/cmd/docker_plugin_apitest.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/jenkins-zh/jenkins-cli/app/i18n"
+	"github.com/jenkins-zh/jenkins-cli/client"
+	jenkinsFormula "github.com/jenkins-zh/jenkins-formulas/pkg/common"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+func init() {
+	rootCmd.AddCommand(dockerPluginAPITestCmd)
+	flags := dockerPluginAPITestCmd.Flags()
+	flags.StringVarP(&ip, "ip", "", "127.0.0.1",
+		i18n.T("The ip address of the computer you want to use"))
+	flags.StringVarP(&port, "port", "", "8081",
+		i18n.T("The port to connect to the jenkins started in docker"))
+	flags.StringVar(&userName, "username", "admin", i18n.T("The username to use when log into jenkins"))
+	flags.StringVar(&token, "token", "", i18n.T("The token used to log into jenkins"))
+}
+
+var ip string
+var port string
+var jenkinsPluginTest jenkinsFormula.CustomWarPackage
+var pluginsWithProblem []string
+var userName string
+var token string
```

Comment:
I prefer do not to expose all these variables. You can put some of them into https://github.com/jenkins-zh/jenkins-cli/pull/624/files#diff-02fba72fba6090f3f60ca286fe093f7744d9fe54dd4a1f006b60b1873c0a5bfaR39